config/v1/tests/infrastructures.config.openshift.io/VSphereMultiVCenters.yaml

```diff
@@ -85,8 +85,18 @@ tests:
                 port: 443
                 server: server2.dev.cluster.com
   onUpdate:
-    - name: Should be able to set vCenters to multiple
+    - name: Should be able to set vCenters to single from empty spec
```

Comment:
What's the use case here?

Reply:
This is in the case user upgrades cluster from a version that did not have vsphere platform spec and wanted to create failure domains.  In this case, you will need to set a vCenter in the vcenters section.